src/message/agent/mmbwmon/system_info.cpp

```diff
@@ -0,0 +1,34 @@
+#include <fast-lib/message/agent/mmbwmon/system_info.hpp>
+
+namespace fast {
+namespace msg {
+namespace agent {
+namespace mmbwmon {
+
+system_info::system_info(const size_t _threads, const size_t _smt, const size_t _numa, const std::vector<double> &_membw): threads(_threads), smt(_smt), numa(_numa), membw(_membw)
+{
+}
+
+YAML::Node system_info::emit() const
+{
+	YAML::Node node;
+	node["task"] = "mmbwmon system info";
```

Comment:
Isn't that redundant information? You already have the task encoded in the topic.